pkg/repository/prisma/step_run.go

```diff
@@ -1921,7 +1921,7 @@ func (s *stepRunEngineRepository) GetStepRunBulkDataForEngine(ctx context.Contex
 	})
 }
 
-func (s *stepRunEngineRepository) ListInitialStepRunsForJobRun(ctx context.Context, tenantId, jobRunId string) ([]*dbsqlc.GetStepRunForEngineRow, error) {
+func (s *sharedRepository) ListInitialStepRunsForJobRun(ctx context.Context, tenantId, jobRunId string) ([]*dbsqlc.GetStepRunForEngineRow, error) {
```

Comment:
```suggestion
func (s *sharedRepository) listInitialStepRunsForJobRun(ctx context.Context, tenantId, jobRunId string) ([]*dbsqlc.GetStepRunForEngineRow, error) {
```

Reply:
This is also accessed from outside the package
JobsControllerImpl
`	startableStepRuns, err := ec.repo.StepRun().ListInitialStepRunsForJobRun(ctx, metadata.TenantId, payload.JobRunId)
`